easybuild/easyconfigs/i/igraph/igraph-0.9.4-foss-2021a.eb

```diff
@@ -0,0 +1,51 @@
+# Author: Denis Krišťák (INUITS)
+# Modified: Jasper Grimm (UoY)
+
+easyblock = 'CMakeMake'
+
+name = 'igraph'
+version = '0.9.4'
+
+homepage = 'https://igraph.org'
+description = """igraph is a collection of network analysis tools with the emphasis on
+efficiency, portability and ease of use. igraph is open source and free. igraph can be
+programmed in R, Python and C/C++."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/igraph/igraph/releases/download/%(version)s']
+sources = ['%(name)s-%(version)s.tar.gz']
+checksums = ['a3285cccf4f043c9ced2bc8d8d2609ff8398cb92ed49fdf86264ed91929137dd']
+
+builddependencies = [
+    ('Autotools', '20210128'),
+    ('pkg-config', '0.29.2'),
+    ('CMake', '3.20.1'),
+]
+
+dependencies = [
+    ('GLPK', '5.0'),
+    ('libxml2', '2.9.10'),
+    ('zlib', '1.2.11'),
+    ('SuiteSparse', '5.10.1', '-METIS-5.1.0'),
+    ('arpack-ng', '3.8.0'),
+]
+
+# Build static and shared libraries
+configopts = ["-DBUILD_SHARED_LIBS=OFF", "-DBUILD_SHARED_LIBS=ON"]
```

Comment:
@jfgrimm could you explain this? Looks weird to disable and enable the same option...

Also, was it deliberate that you removed the previous `configopts = "--with-external-blas --with-external-lapack --with-external-glpk"`?

Reply:
The old configopts are no longer supported (or at least, some of them; didn't exclusively test all). It looks like it correctly picks up and links the EB libraries nonetheless.

Reply:
The build scripts they use don't support building both static and shared libs at the same time. I guess that could be patched, but building twice (once with static, once shared) works without needing a patch

Reply:
Ah, yes - I saw the two iterations. Thanks for confirming.